src/VirtoCommerce.CatalogExportImportModule.Web/Scripts/blades/file-upload.tpl.html

```diff
@@ -62,7 +62,8 @@
                             </div>
                             <div class="d-flex justify-content-end"  ng-if="!(csvMaxSizeError || fileTypeError)">
                                 <span class="btn btn-small btn-danger-outline" ng-class="{'mr-10': !internalCsvError}" ng-click="deleteUploadedItem()">{{ 'catalogExportImport.blades.file-upload.actions.delete' | translate }}</span>
-                                <span class="btn btn-small border-primary" ng-if="!internalCsvError" ng-click="showPreview()">{{ 'catalogExportImport.blades.file-upload.actions.preview' | translate }}</span>
+                                <span class="btn btn-small border-primary" ng-if="!internalCsvError && blade.dataType.value === 'EditorialReview'" ng-click="showPreview()">{{ 'catalogExportImport.blades.file-upload.actions.preview' | translate }}</span>
```

Comment:
Use the real value comparison. For example 'PhisicalProduct'. And move these values to the frontend modules constans.

Reply:
done